autovalet_goal_generation/launch/seg_map.launch

```diff
@@ -0,0 +1,85 @@
+<!--
```

Comment:
so it seems like a lot of the calls in this file could be separated out into its own file. and thne maybe both parking_lot.launch and this file could call that file? idk. could be more trouble when it's worth, i just don't love the idea of copy-pasting code. your call

Reply:
Done. Check it out now!